src/cow-react/modules/limitOrders/containers/LimitOrdersWidget/index.tsx

```diff
@@ -124,52 +126,59 @@ export function LimitOrdersWidget() {
             <TradeWidgetLinks />
             <SettingsWidget />
```

Comment:
@nenadV91 I think we should hide `SettingsWidget` when Limit-orders is not unlocked. What do you think?

Reply:
Good point!